playbooks/tasks/portal-configs-get.yml

```
@@ -0,0 +1,68 @@
+---
+
+# Get portal configs
+# - Load portal configs from secrets backend (e.g. LastPass)
+# - Set default values (for unset keys)
+# - Set mongodb flag
+# - Set mongodb config (if mongodb is on)
+# - Set accounts flag
+
+- name: Include loading portal config
+  include_tasks: "{{ playbook_dir }}/{{ load_portal_config_handler }}"
+
+- name: Set portal config to empty (if it hasn't yet been saved to secrets backend)
+  set_fact:
+    webportal_server_config: "{{ webportal_server_config | default({}) }}"
+
+- name: Set portal config missing variables to default values
+  set_fact:
+    webportal_server_config: "{{ webportal_server_config | combine({item.key: item.value}) }}"
+  when: webportal_server_config[item.key] is not defined
+  loop: "{{ webportal_server_config_defaults | dict2items }}"
+  no_log: True
+
+- name: Set MongoDB flag
+  set_fact:
+    portal_mongodb_on: "{{ webportal_server_config.portal_modules is defined and webportal_server_config.portal_modules is not none and ('m' in webportal_server_config.portal_modules or 'a' in webportal_server_config.portal_modules) }}"
+
+# Get MongoDB config (if MongoDB is on)
+- block:
+    - name: Set MongoDB config keys
+      set_fact:
+        mongodb_config_keys:
+          - skynet_db_user
+          - skynet_db_pass
+
+    - name: Set MongoDB config (server config can override common config)
+      set_fact:
+        mongodb_config: "{{ mongodb_config | default({}) | combine({item: webportal_server_config[item] | default(webportal_common_config[item]) }) }}"
+      no_log: True
+      ignore_errors: True
+      register: mongodb_config_result
+      loop: "{{ mongodb_config_keys }}"
+
+    - name: Fail if there are some missing MongoDB config items
+      fail:
+        msg: |
+          Portal configuration error for MongoDB:
+
+          For MongoDB you must configure all of the following items:
+          {% for key in mongodb_config_keys %}
+          - {{ key }}
+          {% endfor %}
+          in server or common config.
+
+          For these items server config overrides common config if the item is defined in both of them.
+          
+          The following item(s) are not configured:
+          {% for key in mongodb_config_keys %}
+          {% if mongodb_config[key] is not defined %}
+          - {{ key }}
+          {% endif %}
+          {% endfor %}
+      failed_when: mongodb_config_result.failed | default(False)
+  when: portal_mongodb_on
+
+- name: Set Accounts flag
+  set_fact:
+    portal_accounts_on: "{{ webportal_server_config.portal_modules is defined and webportal_server_config.portal_modules is not none and 'a' in webportal_server_config.portal_modules }}"
```

can this be a multiline as well?
```suggestion
    portal_accounts_on: >
      "{{ webportal_server_config.portal_modules is defined and 
      webportal_server_config.portal_modules is not none and 
      'a' in webportal_server_config.portal_modules }}"
```